requestor-tutorials/vm-runtime/computation-payload-manifest.md

```diff
@@ -0,0 +1,283 @@
+        ]
+        ```
+
+      - Mix of both
+
+      Commands `deploy`, `start` and `terminate` are always allowed. These values become the default if no `compManifest.script.commands` property has been set, but the `compManifest` object is present.
+
+    - #### `compManifest.script.match` : String
+    
+      Selects a default way of comparing command arguments stated in the manifest and the ones received in the ExeScript, unless stated otherwise in a command JSON object.
+
+      The match property could be one of:
+
+        - `strict`: byte-to-byte argument equality (default)
+
+        - `regex`: treat arguments as regular expressions
+
+          Syntax: Perl-compatible regular expressions (UTF-8 Unicode mode), w/o the support for look around and backreferences (among others).
+  
+  - #### `compManifest.net` : Object
+
+    Applies constraints to networking.
+
+    - #### `compManifest.net.inet.out` : Object
+
+      Outgoing requests to the public Internet network constraints.
+
+      - #### `compManifest.net.inet.out.protocols` : List[String]
+
+        List of allowed outbound protocols. Currently fixed at ["http", "https"].
+
+      - #### `compManifest.net.inet.out.urls` : List[String]
+
+        List of allowed external URLs that outbound requests can be sent to. E.g. ["https://api.some-public-service.com", "https://some-other-service.com/api/resource"]
+
+#### Example of _Computation Payload Manifest_ with _Computation Manifest_ definition:
+
+```json
+{
+  "version": "0.1.0",
+  "createdAt": "2022-09-15T00:00:00.000000Z",
+  "expiresAt": "2100-01-01T00:00:00.000000Z",
+  "payload": [
+    {
+      "urls": [
+        "http://yacn2.dev.golem.network:8000/docker-golem-script-curl-latest-d75268e752.gvmi"
+      ],
+      "hash": "sha3:e5f5ddfd649525dbe25d93d9ed51d1bdd0849933d9a5720adb4b5810"
+    }
+  ],
+  "compManifest": {
+    "version": "0.1.0",
+    "script": {
+      "commands": [
+        "run curl.*",
+        "transfer .*/output.txt"
+      ],
+      "match": "regex"
+    },
+    "net": {
+      "inet": {
+        "out": {
+          "protocols": ["https"],
+          "urls": ["https://api.some-public-service.com"]
+        }
+      }
+    }
+  }
+}
+```
+
+### Certificates
+
+_App author's certificate_ gets send in a request together with a _Computation Payload Manifest_ and its signature. Certificate is used to verify signature. In order to verify signature _Provider_ first needs to verify incoming _App author's certificate_. To do so it has to have certificate used to sign _App author's certificate_ [imported](#3-importing-application-authors-certificates) into its keystore (together with every intermediate certificate in the chain).
+
+#### Manifest signature
+
+Signature allows _Provider_ to verify content of incoming _Computation Payload Manifest_.
+
+It can be generated using `openssl` tool and a private key related to _App author's certificate_ signed by some trusted by _Providers_ certificate.
```

Comment:
```suggestion
It can be generated using the `openssl` tool and a private key related to _App author's certificate_ signed by some trusted by _Providers_ certificate.
```